BitwardenShared/Core/Vault/Repositories/VaultRepositoryTests.swift

```diff
@@ -585,26 +595,40 @@ class VaultRepositoryTests: BitwardenTestCase { // swiftlint:disable:this type_b
         }
     }
 
+    /// `downloadAttachment(_:cipher:)` throws an error if the cipher can't be found in local data storage.
+    func test_downloadAttachment_cipherNotFound() async throws {
+        await assertAsyncThrows {
+            let attachment = AttachmentView.fixture(fileName: "sillyGoose.txt")
+            let cipherView = CipherView.fixture(
+                attachments: [attachment],
+                id: "2"
+            )
+            _ = try await subject.downloadAttachment(attachment, cipher: cipherView)
+        }
+    }
+
     /// `downloadAttachment(_:cipher:)` updates the cipher on the server if the SDK adds a cipher key.
     func test_downloadAttachment_updatesMigratedCipher() async throws {
```

Comment:
🤔 Does this test still make sense? I think it's almost the same as `test_downloadAttachment()` now that we removed the `encryptAndUpdateCipher()` call. Given that we aren't updating the cipher as part of downloading, I think we could remove it.

Reply:
yeah just removed.